packages/ng-table-builder/src/table-builder.component.ts

```diff
@@ -115,6 +119,34 @@ export class TableBuilderComponent<T>
         return this.sourceRef.length > 0;
     }
 
+    public get rootHeight(): string {
```

Comment:
can we use pipe instead getter?

Reply:
I think not, because I would like to get rootHeight from outside to find out what height the table has.